packages/template/typescript/src/TypeScriptTemplate.ts

```diff
@@ -0,0 +1,51 @@
+import { ForgeTemplate } from '@electron-forge/shared-types';
+import { asyncOra } from '@electron-forge/async-ora';
+
+import fs from 'fs-extra';
+import path from 'path';
+
+const copyTemplateFile = async (destDir: string, basename: string) => {
+  const templateDir = path.resolve(__dirname, '..', 'tmpl');
+  await fs.copy(path.join(templateDir, basename), path.resolve(destDir, basename));
+};
+
+class TypeScriptTemplate implements ForgeTemplate {
+  public devDependencies = [
+    'typescript@^3.7.0',
+    'tslint@^5.20.0',
```

Comment:
What do you think about using ESLint + the `@typescript-eslint/eslint-plugin` instead of `tslint`?

Reply:
works for me! i'm totally unopinionated about that specifically :)

Reply:
in looking at this more - using eslint with base rules means adding a decent number of dependencies; is there a reason you prefer it? tslint is a bit simpler and more straightforward to add via template i would say

Reply:
I mostly prefer it because otherwise you can't use other things in the ESLint ecosystem (e.g., `eslint-plugin-mocha` to detect `.only`). But in this case, it's probably better to have a simple dependency.